entrypoint.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/bash
-
-git config --global --add safe.directory /github/workspace
-
-args=("$@")
-command="python /app/mpyl/__main__.py"
-
-for arg in "${args[@]}"
-do
-    # Split the argument on spaces and iterate over the words
```

Comment:
how do you fix this problem now? Because you always know each input we never get passed multiple words in 1 input right?

Reply:
By passing multiple `args` instead of a single one:
❌
```yaml
args:
    - build run --ci --stage ${{ inputs.stage }} --projects ${{ inputs.project }}
```
✅
```yaml
args:
    - build 
    - run 
    - --ci 
    - --stage 
    - ${{ inputs.stage }} 
    - --projects 
    - ${{ inputs.project }}
```

Reply:
This PR is still on draft because there's a small issue with the `--verbose` flag, Github is kinda stupid and converts `null` to an empty string `''`, which then click rejects because `''` is not a valid parameter.
Should be fixable but I haven't gotten around to it yet 🙂

Reply:
> By passing multiple args instead of a single one

Yeah but this wasn't possible before for some reason.. we looked at it together. I don't remember the details, something with how python receives the args? Anyway, great if you fixed it.

Reply:
I also don't remember it 🤷  

Anyway, there's no good way to address the last issue I had pending (described above), so I made it easy on ourselves: instead of converting the `RUNNER_DEBUG` environment variable to a CLI `--verbose` input parameter, we now simply read the environment variable inside MPyL. [Seems to work well](https://github.com/Vandebron/gh-mpyl/actions/runs/11755844047/job/32751389368?pr=33#step:5:80) 👍

Reply:
Meh, it's kinda the opposite of what we were trying to achieve in other places with using input/cli params instead of reading from environment variables. But if it's the only way then that's fine!

Reply:
Yes, it kinda is. There is a tiny difference though: it's _already_ an environment variable (we're not setting it ourselves like `MPYL_CONFIG_PATH` or `TAG`), so that's why I'm a bit less worried about it.

Reply:
Yeah it's a bit more unclear than `-v` when using the cli outside the context of github actions, but that's okay.